addons/website/static/src/js/tours/tour_utils.js

```diff
@@ -322,55 +322,55 @@ function clickOnExtraMenuItem(stepOptions, backend = false) {
  * @param {() => TourStep[]} steps The steps of the tour. Has to be a function to avoid direct interpolation of steps. 
  */
 function registerWebsitePreviewTour(name, options, steps) {
-    if(typeof steps !== "function") {
-        throw new Error(`tour.steps has to be a function that returns TourStep[]`)
+    if (typeof steps !== "function") {
+        throw new Error(`tour.steps has to be a function that returns TourStep[]`);
     }
-    return registry
-        .category("web_tour.tours")
-        .add(name, {
-            ...options,
-            url : getClientActionUrl(options.url, !!options.edition), 
-            steps: () => {
-                const tourSteps = [...steps()];
-                // Note: for both non edit mode and edit mode, we set a high timeout for the
-                // first step. Indeed loading both the backend and the frontend (in the
-                // iframe) and potentially starting the edit mode can take a long time in
-                // automatic tests. We'll try and decrease the need for this high timeout
-                // of course.
-                if (options.edition) {
-                    tourSteps.unshift({
-                        content: "Wait for the edit mode to be started",
-                        trigger: '.o_website_preview.editor_enable.editor_has_snippets',
-                        timeout: 30000,
-                        auto: true,
-                        run: () => {}, // It's a check
-                    });
-                } else {
-                    tourSteps[0].timeout = 20000;
-                }
-                return tourSteps;
+    return registry.category("web_tour.tours").add(name, {
+        ...options,
+        url: getClientActionUrl(options.url, !!options.edition),
+        steps: () => {
+            const tourSteps = [...steps()];
+            // Note: for both non edit mode and edit mode, we set a high timeout for the
+            // first step. Indeed loading both the backend and the frontend (in the
+            // iframe) and potentially starting the edit mode can take a long time in
+            // automatic tests. We'll try and decrease the need for this high timeout
+            // of course.
+            if (options.edition) {
+                tourSteps.unshift({
+                    content: "Wait for the edit mode to be started",
+                    trigger: ".o_website_preview.editor_enable.editor_has_snippets",
+                    timeout: 30000,
+                    auto: true,
+                    run: () => {}, // It's a check
+                });
+            } else {
```

Comment:
While this whole diff? Seems like it was just missing a semicolon, had 2 extra whitespaces and a missing space, no?

Reply:
I had change all of this lines in previous commit. 
Eslint requires this in only one line 
`return registry.category("web_tour.tours").add(name, {`
So the next lines needs new indentation.

Reply:
`eslint` doesn't require that

![image](https://github.com/odoo/odoo/assets/30048408/f1f83512-a925-4d5e-9bbf-4a190726e170)

I guess the framework-js specific config does.. We don't use that config ourselves but it's fine, thanks for the explanation

Reply:
I don't think so ... but it should be interesting to know why we have differences.

Reply:
```js
    return registry
        .category("web_tour.tours")
        .add(name, {
            // ...
        });
```
Is definitely valid eslint wise, it's just that the eslint config of the framework-js team is using prettier on top of eslint.
That's why the difference.
Note that we don't use it in website, we decided it doesn't bring much value, eslint is enough for us.
But it's fine, we can do an exception here even if it's useless diff